src/alto-ui/Aside/Aside.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { bemClass } from '../helpers/bem';
+
+import CloseButton from '../CloseButton';
+
+import './Aside.scss';
+
+function Aside({ className, onClose, show, title, children, a11yCloseLabel }) {
+  if (!show) return null;
+  return (
+    <aside className={bemClass('Aside', {}, className)}>
```

Comment:
classnames ?

Reply:
yes, I usually use `bemClass()` so it's ready to be used. As you prefer.